src/zenml/cli/base.py

```diff
@@ -54,40 +61,96 @@
 @click.option(
     "--template",
     is_flag=True,
-    help="Use a ZenML project template to initialize the repository.",
+    default=False,
     required=False,
+    help="Use the ZenML starter project template to initialize the repository "
+    "and prompt the user to enter parameter values for the template.",
+)
+@click.option(
+    "--starter",
+    is_flag=True,
     default=False,
+    required=False,
+    help="Use the ZenML starter project template to initialize the repository.",
 )
-def init(path: Optional[Path], template: bool = False) -> None:
+def init(
+    path: Optional[Path],
+    template: bool = False,
+    starter: bool = False,
+) -> None:
     """Initialize ZenML on given path.
 
     Args:
         path: Path to the repository.
-        template: Whether to use a ZenML project template to initialize the
-            repository.
+        template: Whether to use the ZenML starter project template to
+            initialize the repository and prompt the user for parameter values.
+        starter: Whether to use the ZenML starter project template to
+            initialize the repository.
     """
     if path is None:
         path = Path.cwd()
 
-    if template:
+    if template or starter:
         try:
-            from copier.cli import CopierApp
+            from copier import Worker
         except ImportError:
             error(
                 "You need to install the ZenML project template requirements "
                 "to use templates. Please run `pip install zenml[templates]` "
                 "and try again."
             )
             return
-        declare(
-            "Next, you will be prompted to generate a project from the "
-            "template."
-        )
-        CopierApp.run(
-            ["copier", "gh:zenml-io/zenml-project-templates", str(path)],
-            exit=False,
+
+        from zenml.cli.text_utils import (
+            zenml_go_privacy_message,
+            zenml_go_welcome_message,
```

Comment:
```suggestion
            zenml_cli_privacy_message,
            zenml_cli_welcome_message,
```

Probably wants changing? (and in `text_utils.py` file)